public/js/controllers/dashboard.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable */
+angular.module('mean.system')
+    .controller('dashboardController', ['$scope', 'Global', '$http', '$q', '$window',
+        ($scope, Global, $http, $q, $window) => {
+            $scope.global = Global;
+            $scope.pageNumber = 1;
+            $scope.pagination = null;
+            $scope.leaderboardPage = 1;
+            $scope.gameLogPage = 1;
+            $scope.showGameOptions = false;
+
+            const token = localStorage.getItem("card-game-token");
+
+            $scope.getGameLog = (page) => {
+                $scope.showTab1 = true;
+                $scope.showTab3 = false;
+                $scope.showTab2 = false;
+                $scope.gameLogPage = page;
```

Comment:
Good work so far. Take your variable naming into consideration so that they convey the meaning of tasks to be done